experiments/samples/world-cultures-geography/fileset/package.opf

```diff
@@ -0,0 +1,97 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<package xmlns="http://www.idpf.org/2007/opf" prefix="dc: http://purl.org/dc/elements/1.1/" version="3.0"
+	unique-identifier="d21041e30">
+	<metadata xmlns:dc="http://purl.org/dc/elements/1.1/">
+		<!-- required metadata -->
+		<dc:creator>Sarah Witham Bednarz</dc:creator>
+		<dc:creator>Ines M. Miyares</dc:creator>
+		<dc:creator>Mark C. Schug</dc:creator>
+		<dc:creator>Charles S. White</dc:creator>
+		<dc:format>eBraille 1.0</dc:format>
+		<dc:identifier id="d21041e30">41f1328c-0571-4e71-8be8-e65bc148281a</dc:identifier>
+		<dc:language>en-Brai-US</dc:language>
+		<dc:title id="d20201e20">World Cultures and Geography</dc:title>
+		<dc:title id="d20201e1149">Legends of Landforms</dc:title>
+		<dc:title id="d20201e1158">Why Snails Have Shells</dc:title>
+		<meta property="dcterms:dateCopyrighted">2024</meta>
+		<meta property="dcterms:modified">2015-02-24T13:37:19Z</meta>
+		<meta property="a11y:cellType">8</meta>
+		<meta property="a11y:code">UEB</meta>
```

Comment:
Should this be cellType 6? I don't think there's a 8-dot UEB code. I think we also need to specify contracted or uncontracted.

Reply:
Ugh, yes, I copied the tag and wasn't thinking about the value!

The code value is also technically invalid since the text isn't formatted to any code at this point. I was just making sure all the required metadata was present.

Reply:
Actually I did see some 8-dot cells, e.g. in ⠄⡳⠭⠆⠴⠆⠆⠄: this is a Liblouis thing, it's not UEB. It is a kind of escape sequence that was inserted for a character that was unknown to Liblouis.

Reply:
(in this case it was the character "•" (U+2022))

Reply:
@mattgarrish What do you mean by "text isn't formatted to any code"?